indico/modules/rb/client/js/common/bookings/BookingEdit.jsx

```diff
@@ -218,9 +219,12 @@ class BookingEdit extends React.Component {
             </Grid.Column>
             <Grid.Column stretched>
               {timelineError ? (
-                <div>
-                  <Message icon="times circle outline" error content={timelineError} />
-                </div>
+                <>
+                  <div>
+                    <Message icon="times circle outline" error content={timelineError} />
+                  </div>
+                  <DummyTimeline rowsCount={10} />
```

Comment:
What about `rowCount`, `numRows` or `rows`? IMHO `rowsCount` sounds weird with the plural

Reply:
`rows` works fine.